index.html

```diff
@@ -19,9 +19,9 @@
             </ul>
                 <img class="ham" src="img/bars.svg"/>
             <div class="icons-container">
-                <a href="https://github.com/HazDuck"><img src="img/github.png" class="icon"/></a>
-                <a href="https://twitter.com/peteheyes"><img src="img/twitter.png" class="icon"/></a>
-                <a href="https://uk.linkedin.com/in/pete-heyes-b7b98985"><img src="img/in.png" class="icon"/></a>
+                <a href="https://github.com/HazDuck" target="_blank"><img src="img/github.png" class="icon"/></a>
+                <a href="https://twitter.com/peteheyes" target="_blank"><img src="img/twitter.png" class="icon"/></a>
+                <a href="https://uk.linkedin.com/in/pete-heyes-b7b98985" target="_blank"><img src="img/in.png" class="icon"/></a>
```

Comment:
Instead of the 'icon' classes added to all of these, you could save using the class and instead target the images with '.icons-container img' (I think... 90% sure)